cards-examples/Web/Web-card-article.md

```diff
@@ -10,229 +10,249 @@ path: /
 
+const selector = '.mdc-button, .mdc-icon-button, .mdc-card__primary-action';
+const ripples = [].map.call(document.querySelectorAll(selector), function(el) {
+  return new MDCRipple(el);
+});
 ```
-<source code here>
-The source code example should display as per the interactive example (https://material.io/components/cards/#) with supporting text and Buttons:
-* Display two elevated cards, for each card
-* Display a card title "Card title 1" for one card and "Card title 2" for the other
-* Display a secondary title "Secondary text" with an opacity of 60%
-* Display text reading "Greyhound divisively hello coldly wonderfully marginally far upon excluding." with an opacity of 60%
-* Display two actions, "Action 1" and "Action 2" with two text buttons
-* Display the sample images ![sample card image of yellow and red tulips](assets/card-sample-image.jpg) and ![sample card image of red and yellow apples](assets/card-sample-image-2.jpg)
-* Allow the cards to be moveable.
-```
-
-### Key properties
-
-![Card anatomy diagram](assets/card-anatomy.png)
 
-**1. Elevated card attributes**
+> _NOTE_: If your card includes any [icon button toggles](../mdc-icon-button#icon-button-toggle), you will want to
+> instantiate `MDCIconButtonToggle` instead of `MDCRipple`.
 
-1. **Container** 
-2. **Thumbnail [optional]** 
-3. **Header text [optional]** 
-4. **Subhead [optional]** 
-5. **Media [optional]** 
-6. **Supporting text [optional]** 
-7. **Buttons [optional]** <
-8. **Icons [optional]** 
+## Card 
 
-<b>Container</b>
-
-Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Container fill color| |
-Container ink color | |
-Container shape radius | |
-Container outline color | |
-Container outline width | | 
-Container horzontal padding | | 
-Container elevation | |
+### Card example
 
-<b>Thumbnail (optional)</b>
+```html
+<div class="mdc-card">
+  <!-- ... content ... -->
+</div>
+```
 
+**Note**: MDC Card is designed to accommodate a wide variety of use cases. See the [Card Contents](#card-contents) section for information on helpers for specific types of content within cards.
 
-|  | Attribute | Related method(s) | Default value |
-|---|---|---|---|
-|Desc. 1 | | | |
+## Other Variants
 
-<b>Header text (optional)</b>
+### Outlined Card
 
-Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Text label | |  
-Text color | |
-Typography | |
+By default, cards are elevated with no outline. You can render unelevated outlined cards instead by adding the
+`mdc-card--outlined` modifier class.
 
-<b>Subhead (optional)</b>
+```html
+<div class="mdc-card mdc-card--outlined">
+  <!-- ... content ... -->
+</div>
+```
 
-Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Text label | |  
-Text color | |
-Typography | |
+### Card Contents
 
-<b>Media (optional)</b>
+MDC Card can be used for a wide variety of use cases, but it includes styles for a few common ones.
 
+#### Primary Action
```

Comment:
Use sentence case for titles: capitalize the first word, and only capitalize following proper nouns.